app/javascript/components/top-corner-controller.jsx

```diff
@@ -26,7 +26,7 @@ const Row = styled.div`
   margin-right: 100px;
 `;
 
-const TopCorner = ({ showWeather }) => {
+const TopCorner = props => {
```

Comment:
Its idiomatic to destructure in the argument list like: `({ showWeather, location })`